docs/core/compatibility/sdk/6.0/outputtype-not-set-automatically.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Breaking change: `OutputType` no longer automatically set to WinExe"
+description: Learn about the breaking change in .NET 6 where `OutputType` for WPF and Windows Forms projects is no longer automatically set to `WinExe`.
+ms.date: 08/17/2021
+---
+# OutputType not automatically set to WinExe
```

Comment:
This may be a bit clearer about the scope of the change:

```suggestion
# OutputType not changed from Exe to WinExe for Windows Forms and WPF projects
```